src/screens/mentee-home/components/avatar-header/avatar-header.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import { AppRoute, HeadingLevel } from 'src/common/enums';
+import { Heading } from 'src/components';
+import { Avatar } from '..';
+import styles from './styles';
+
+interface Props {
```

Comment:
Rewrite all props in such way as it's done in other components
```suggestion
type HeaderProps {
```